suite/cstest/src/test_case.c

```diff
@@ -75,11 +75,12 @@ char *test_input_stringify(const TestInput *test_input, const char *postfix)
 		}
 	}
 	str_append_no_realloc(opt_seq, sizeof(opt_seq), "]");
-	cs_snprintf(msg, msg_len,
-		    "%sTestInput { arch: %s, options: %s, addr: 0x%" PRIx64
-		    ", bytes: %s }",
-		    postfix, test_input->arch, opt_seq, test_input->address,
-		    byte_seq);
+	cs_snprintf(
+		msg, msg_len,
+		"%sTestInput { name: %s, arch: %s, options: %s, addr: 0x%" PRIx64
+		", bytes: %s }",
+		postfix, test_input->name, test_input->arch, opt_seq,
```

Comment:
```suggestion
		prefix, test_input->name, test_input->arch, opt_seq,
```

Please rename while you are at it.

Reply:
Updated.